package.json

```diff
@@ -12,6 +12,9 @@
   "license": "Apache Version 2.0",
   "dependencies": {
     "express": "^4.12.0",
-    "socket.io": "~1.0"
+    "socket.io": "~1.0",
```

Comment:
Lets make these ```"socket.io": "^1.2"```, same as the client https://github.com/uProxy/freedom-social-quiver/blob/master/package.json#L43